guides/common/modules/proc_creating-a-playbook-with-modules-from-project-ansible-collections.adoc

```diff
@@ -0,0 +1,124 @@
+[id="creating-a-playbook-with-modules-from-{project-context}-ansible-collections"]
+= Creating a playbook with modules from {Project} Ansible Collections
+
+Ansible modules from the {Project} Ansible Collections must be able to communicate with the {Project} API over HTTPS.
+In your playbooks, include parameters specifying how to authenticate to enable the connection to API.
+
+[IMPORTANT]
+====
+Do not store sensitive credentials, such as username and password, directly in your playbooks or environment variables.
+The following examples use Ansible vault to manage sensitive data.
+====
+
+.Prerequisites
+* A user account in {Project} exists with permissions to perform the action defined in your playbook.
+* You have access to one of the following types of authentication credentials for that user:
+** Username and password
+** Kerberos ticket, if your {ProjectServer} is configured to use {FreeIPA} as an external authentication source
+** Username and {Project} Personal Access Token
```

Comment:
```suggestion
** Username and password
** Username and {PAT}
** Kerberos ticket, if your {ProjectServer} is configured to use {FreeIPA} as an external authentication source
```

* the sorting felt more natural this way -- no idea why
* there is an attribute for PAT (why?) and the {Project} in front seems duplicating -- you're not saying {Project} password either.